test/common/testinfra.py

```diff
@@ -149,6 +149,30 @@
     'TEST_DIR',
 )
 
+def determine_github_base():
+    # pick a base
+    # see if we have a git checkout - it can be in https or ssh format
+    formats = [
+        re.compile("""origin\thttps:\/\/github\.com\/(.*) \(fetch\)"""),
+        re.compile("""origin\tgit@github.com:(.*)\.git \(fetch\)""")
+        ]
+
+    try:
+        remote_output = subprocess.Popen(["git", "remote", "-v"], stdout=subprocess.PIPE, cwd=TEST_DIR).communicate()[0]
```

Comment:
Right. I should have been more clear. But I guess I was wondering why we don't look up where "master" is coming from, rather than assuming "origin" is the right one to look up.